src/components/payments/registered/RegisteredAppFee.vue

```diff
@@ -0,0 +1,186 @@
+                                            <option value=''>YYYY</option>
+                                            <option value='19'>2019</option>
+                                            <option value='20'>2020</option>
+                                            <option value='21'>2021</option>
+                                        </select> 
+                                    </div>
+                                    <p class="dcww-field__validation">Validation message</p>
+                                </div>
+                            </div>
+                            <div class="dcww-fieldset__row">
+                                <div class="dcww-field">
+                                    <label class="dcww-field__label">CV2</label>
+                                    <input type="number" class="dcww-field__input dcww-field__input--mini" placeholder="CV2" maxlength="10">
+                                    <p class="dcww-field__validation">Validation message</p>
+                                </div>
+                            </div>                        
+                        </div>
+                        <div class="dcww-fieldset__actions">
+                            <input type="button" value="Back" class="dcww-button dcww-button--secondary dcww-button--left" />
+                            <input type="button" value="Pay" class="dcww-button dcww-button--primary dcww-button--chevron dcww-button--right" />
+                        </div>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </main>
+
+    <footer class="dcww-payments-footer">
+      <div class="dcww-legal-copy">
+          <p class="dcww-legal-copy__text">
+              Dŵr Cymru Cyf, a limited company registered in Wales No. 2366777. Registered office: Pentwyn Road, Nelson, Treharris, Mid Glamorgan CF46 6LY.
+          </p>
+          <p class="dcww-legal-copy__copyright">© Dŵr Cymru Cyf 2019</p>
+      </div>
+    </footer>
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: "RegisteredAppFee"
+};
+</script>
+<style>
+
+/* Chrome, Safari, Edge, Opera */
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+/* Firefox */
+input[type=number] {
+  -moz-appearance:textfield;
+}
+
+.dcww-field__dropdown {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    background: #fff url('../../../assets/chevron-down.svg') no-repeat 95% 50%;
+    background-size: 32px;
+    border-radius: 4px;
+    -webkit-box-shadow: 0 0 0 1px #65858f;
+    box-shadow: 0 0 0 1px #65858f;
+    border: 0;
+    min-height: 60px;
+    margin-left: 10px;
+    padding: 18px 35px 18px 20px;
+    width: 100%;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-weight: 300;
+    color: #09333f;
+    line-height: normal;
+}
```

Comment:
does this dropdown stuff need repeating?

Reply:
Good catch - removed